test/api/games.js

```diff
@@ -0,0 +1,44 @@
+const expect = require("chai").expect;
+const mongoose = require('mongoose');
+const request = require("superagent");
+const dataMakers = require("../helpers/data");
+
+describe("api/games", ()=>{
+  it("should receive a list of all games with a production release, without requiring a token", done => {
+    dataMakers.makeGame("prod")
+    .then(()=> {
```

Comment:
I have mixed feels on this. Generally I try to avoid using `await` as it forces the function to be blocking, but maybe that is a good thing in a e2e situation?

Reply:
I think I'm going to do this refactor, along with some others making the dataMakers stuff more general purpose, and also the 'standard' way of creating objects for these tests. As for the async/await blocking topic, I'm personally for forcing blocking when I can (still a python dev at heart 😛 ).

Reply:
I think this is _especially_ a good use of `await` since it'll allow us to wait for UI elements to be ready. Almost everything in `selenium-webdriver` is promise-based, and so `await` will just allow us to not have tons of `then` calls that muddy up the code and obscure the intention of the test (IMO)

Reply:
Either way is fine with me